meta-mender-update-modules/recipes-mender/mender-update-modules/mender-ipk_git.bb

```diff
@@ -0,0 +1,18 @@
+DESCRIPTION = "The IPK Update Module installs opkg-based package in the target."
+
+require mender-update-modules.inc
+
+inherit allarch
+
+python do_configure() {
+    if not bb.utils.contains('PACKAGE_CLASSES', 'package_ipk', True, False, d) or \
+       not bb.utils.contains('IMAGE_FEATURES', 'package-management', True, False, d):
+           bb.warn("Installing ipk update module without on-target IPK package management.")
```

Comment:
I would generate an error here. No point in installing if it is not usable.

I would also update the error message to include instructions on how to fix it, e.g by setting `PACKAGE_CLASSES` and `IMAGE_FEATURES` to appropriate values.